lighthouse-cli/test/cli/run-test.js

```diff
@@ -19,42 +19,73 @@ const fastConfig = {
   },
 };
 
+// Map plugin name to fixture since not actually installed in node_modules/.
+jest.mock('lighthouse-plugin-simple', () => {
+  return require('../../../lighthouse-core/test/fixtures/config-plugins/lighthouse-plugin-simple');
```

Comment:
`.js` of the real file?

Reply:
> or we are making it work like a package so it's ok?

I was making it like a package (I added a `/` to make it clear it's a directory), but can also make it the js file if you'd rather

Reply:
nah, it's cool